src/AhRulesBot/AhRulesBot/CustomRulesImport/CustomRulesWorker.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Extensions.Hosting;
+using Serilog;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using AhRulesBot.Models;
+using System.IO;
+using System.Collections.Generic;
+using AhRulesBot.Infrastructure;
+using System.Linq;
+using System.Globalization;
+using CsvHelper;
+using System.Threading.Channels;
+using AhRulesBot.CustomRulesImport;
+
+namespace AhRulesBot.OldMessagesProcessing
+{
+    internal class CustomRulesWorker : BackgroundService
+    {
+        private readonly ILogger _logger;
+        private readonly RulesDriveService _rulesDriveService;
```

Comment:
why do you need wrapper, if you do not hide it behind interface later?

Reply:
Thank you! Fixed.